Core/Src/main.c

```diff
@@ -87,14 +91,39 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_USART2_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
 
+  //default pulse 3200, 5% of 64000
+  setTimerPulseValue(3200);
```

Comment:
use the `PULSE_RANGE` macro here instead of a magic number. move the comment into the header where `PULSE_RANGE` is defined

Reply:
Addressed :)